### ISSUE TITLE
Indent does not handle JSX fragment

### ISSUE THREAD

**Issue by golopot** [eslint/eslint#12208]
**What rule do you want to change?**
indent
**Does this change cause the rule to produce more or fewer warnings?**
more
**How will the change be implemented? (New option, new default behavior, etc.)?**
New option or new default behavior.
**Versions**
eslint 6.3.0
**Parser**
espree
**Please provide some example code that this change will affect:**

```js
/* eslint indent: [2, 2] */

function Foo () {
  return (
<>
{' '}
</>
  )
}
```

https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IGluZGVudDogWzIsIDJdICovXG5cbmZ1bmN0aW9uIEZvbyAoKSB7XG4gIHJldHVybiAoXG48PlxueycgJ31cbjwvPlxuICApXG59XG4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjksInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnsianN4Ijp0cnVlfX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19

**What does the rule currently do for this code?**
Does not handle indent of anything in `JSXFragment`. In contrast indents in normal jsx elements are handled properly.
**What will the rule do after it's changed?**
Handle indents in jsx fragments.
**Are you willing to submit a pull request to implement this change?**
No.

**Comment by kaicataldo** [comment]
I've verified this and think we should treat it as a bug. I'm working on this!
